kbase-extension/static/kbase/js/widgets/common/LoadingMessage.js

```diff
@@ -0,0 +1,24 @@
+define(['jquery'], ($) => {
+    'use strict';
+    function $LoadingMessage(message) {
+        const $message = (() => {
+            if (typeof message === 'string') {
+                return $('<span>').text(message);
+            } else {
+                return message;
+            }
+        })();
+        return $('<div>')
+            .addClass('alert alert-info')
+            .css('display', 'flex')
+            .css('flex-direction', 'row')
+            .css('align-items', 'center')
+            .css('justify-content', 'center')
+            .css('margin', '10px auto')
+            .css('max-width', '30em')
```

Comment:
move to CSS file

Reply:
Moved to a stylesheet in this and other cases.
I resisted (I have of course done this dozens of times, so I'm quite familiar with this practice!) in this case I considered the risk of css class collision or interference not unlikely in a large codebase without some sort of namespacing solution. When individual components have associated class files, the stylesheet proliferation is be significant.  It is just so easy to invent a new component, an associated class named after it, and have it collide with a same-named component either at present or in the future. There are ways to fix this, like css modules, or namespacing class names.